ext/rugged/rugged_tree.c

```diff
@@ -487,8 +487,8 @@ void rugged_parse_merge_options(git_merge_options *opts, VALUE rb_options)
 			}
 		}
 
-		if (RTEST(rb_hash_aref(rb_options, CSTR2SYM("renames")))) {
-			opts->flags |= GIT_MERGE_FIND_RENAMES;
+		if (rb_hash_aref(rb_options, CSTR2SYM("renames")) == Qfalse) {
```

Comment:
Ah of course.  Sorry, I now realize that my prior suggestion doesn't make sense.  😀